### PR TITLE
pythonPackages.levenshtein: fix 0.18.1 build

### PR DESCRIPTION
**Pull request by yuuyins**
###### Description of changes

won't build after https://github.com/NixOS/nixpkgs/commit/7ed8ac06d1aae3ae78303342b4271b6ff6d968cd

```
substitute(): ERROR: file 'setup.cfg' does not exist
```

cc @mweinelt @fabaff 

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by yuuyins** [comment]
https://github.com/NixOS/nixpkgs/pull/158881 won't build and is blocking

File: pkgs/development/python-modules/rapidfuzz-capi/default.nix
Comment (dotlambda) [inline pkgs/development/python-modules/rapidfuzz-capi/default.nix:24]: nope

File: pkgs/development/python-modules/rapidfuzz-capi/default.nix
Comment (dotlambda) [inline pkgs/development/python-modules/rapidfuzz-capi/default.nix:18]: This also needs `pytestCheckHook` in `checkInputs` or a `checkPhase`.

File: pkgs/development/python-modules/levenshtein/default.nix
Comment (dotlambda) [inline pkgs/development/python-modules/levenshtein/default.nix:63]:
```suggestion
```

File: pkgs/development/python-modules/levenshtein/default.nix
Comment (dotlambda) [inline pkgs/development/python-modules/levenshtein/default.nix:41]: can be patched out or just left out if you use `format = "setuptools"`

File: pkgs/development/python-modules/levenshtein/default.nix
Comment (dotlambda) [inline pkgs/development/python-modules/levenshtein/default.nix:74]:
```suggestion
```
that's automatic for Python packages

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/rapidfuzz-capi/default.nix
Comment (yuuyins) [reply pkgs/development/python-modules/rapidfuzz-capi/default.nix:24]: @dotlambda ? https://github.com/NixOS/nixpkgs/pull/168580/commits/28b65c3cb02f91a37c71eb4baf76acf480fde848#diff-7051ba4a27a1aa6bb264875507a22237c6aaf3566a13335c6c96f76593be5314

i'm probably going to remove this file and wait for your pr to get merged

Comment (dotlambda) [reply pkgs/development/python-modules/rapidfuzz-capi/default.nix:24]: Oops, I meant to put this on the respective line in rapidfuzz-cpp

**Comment by SuperSandro2000** [comment]
already part of python-updates. Feel free to add yourself as maintainer there.

**Pull request closed without merging**
